lib/travis/build/script/csharp.rb

```diff
@@ -48,7 +61,7 @@ def export
         end
 
         def install
-          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution]
+          sh.cmd "nuget restore #{config[:solution]}", retry: true if config[:solution] && !is_mono_2?
         end
 
         def script
```

Comment:
Are we talking about the same thing? My comment was targeted at https://github.com/Joshua-Anderson/travis-build/blob/8a124e510cd40242649fe895a0f0a3a64e8e8d24/lib/travis/build/script/csharp.rb#L70 but GitHub wouldn't let me comment on that line in this PR :)